snmp_base_block.py

```diff
@@ -33,7 +33,7 @@ class SNMPBase(LimitLock, Block):
     """
 
     agent_host = ExpressionProperty(title="SNMP Agent Url", default='127.0.0.1')
-    agent_port = IntProperty(title="SNMP Agent Port", default=161)
+    agent_port = ExpressionProperty(title="SNMP Agent Port", default='161')
```

Comment:
@mattdodge Do you think it's ok to change the default property like this? N.io should be able to take blocks that are configured with an int and load it up as a string in the ExpressionProperty, right?